src/Simulation/QCTraceSimulator/QubitMetrics.cs

```diff
@@ -8,14 +8,27 @@ namespace Microsoft.Quantum.Simulation.QCTraceSimulatorRuntime
     /// </summary>
     public class QubitTimeMetrics
     {
-        // TODO: Qubit Ids are already available in qubits, but DepthCounter doesn't have access to it
+        // Qubit Ids are already available in qubits, but DepthCounter doesn't have access to it
         // in OnPrimitiveOperation because it's not part of IQCTraceSimulatorListener interface.
         // Consider changing architecture to pass qubits rather than metrics in IQCTraceSimulatorListener.
         public long QubitId { get; }
 
+        // If qubit is not fixed in time, this is depth of single-qubit-gate chain on this qubit
```

Comment:
It's not clear what it means for a qubit to be (or not to be) fixed in time.

Reply:
It's not about primary/secondary component. I updated the comment, thanks for some wording.

Reply:
My point was that in the context of this file it's not clear what it means for the qubit to be "fixed or not fixed in time". Maybe refer to the whitepaper doc that explains it?

Reply:
Also, would it be possible to change the whitepaper from .doc to .md? It would be easier to provide feedback on it in this PR and later consume it directly via the github interface.

Reply:
Wait until I push the updated comment. It should be better.

Reply:
I'm reluctant to converting .doc to .md. It has formulas and embedded images. I had troubles with both before. I'll check if they work here.

Reply:
Images can be checked-in alongside the md file and linked to using relative links. Don't know how to do formulas, but yours aren't complex, are they?

Reply:
btw, the updated comment is perfect :)